nav2_amcl/src/amcl_node.cpp

```diff
@@ -864,7 +864,6 @@ AmclNode::laserReceived(sensor_msgs::msg::LaserScan::ConstSharedPtr laser_scan)
     if (lasers_update_[laser_index]) {
       motion_model_->odometryUpdate(pf_, pose, delta);
     }
-    force_update_ = false;
```

Comment:
Also does it mean that particles will be resampled even when the robot is standing still? 
There's tiny amount of movement in Odom when robot is still and particles may diverge over time because of it (without scan matching pulling it together).. we don't want that..?